src/resources/postcss/tribe-events-single-blocks-reskin.pcss

```diff
@@ -49,3 +49,182 @@
 .tribe-theme-genesis .tribe-block__event-website a {
 	font-size: 1.65rem;
 }
+
+/* = Venue block view styles
+============================================= */
+.tribe-block__venue {
+	border-top: 1px solid #d5d5d5;
+	flex-direction: column-reverse;
+	justify-content: flex-start;
+	padding: 22.5px 0;
+
+	@media screen and (min-width: 768px) {
+		flex-direction: row;
+		padding: 32px 0;
+	}
+
+	&.tribe-block__venue--has-map {
+
+		.tribe-block__venue__meta,
+		.tribe-block__venue__map {
+			flex-basis: 100%;
+			width: 100%;
+
+			@media screen and (min-width: 768px) {
+				flex-basis: 49%;
+				width: 49%;
```

Comment:
does 50% not work? what happens if you set `flex: none` rather than the `flex-basis`?

Reply:
I retained the same values that were originally there in the block for desktop. I only needed to update the mobile version.  I've made the updated you recommended and it works fine as well.

Reply:
let's move `flex: none` to the mobile and remove `flex-basis`.

Reply:
Updated